.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      golang-x:
+        patterns:
+          - "golang.org/x/*"
+  - package-ecosystem: "github-actions"
```

Comment:
Regarding this we need to keep in mind that we are primarily a 'library package' and one which is a direct dependency for containerd and CRI-O. The general consideration regarding bumping verisons, apart from our sample plugins, are different for us than for packages which primarily generate standalone binaries. We should not greedily/eagerly bump packages which are in the intersection of our, containerd's or cri-o's direct dependencies, not above what is the current the oldest/lowest version in the union of cri-o and containerd.

Reply:
Makes sense. Closing this PR as automatic dependency updates may not be suitable for a core library that containerd and CRI-O depend on.

Reply:
@ErikJiang Sorry if my initial comment came through as discouraging or unclear.

I didn't mean to say that there is no value or point in dependabot (or imply that the PR should be necessarily closed). I merely tried to describe why I think we can't just go ahead and always bump all of our dependencies unconditionally to the latest available versions (and how much SNR would we then get with that configuration). And I was also wondering (as I'm not familiar with configuring dependabot itself) if we could, for instance, limit the scope first. Maybe start with security alerts...